src/libs/ajax.js

```diff
@@ -641,6 +641,20 @@ export const User = {
     const url = `${await getApiUrl()}/api/user/me/dac/datasets`;
     const res = await axios.get(url, Config.authOpts());
     return res.data;
+  },
+  getAcknowledgements: async () => {
+    const url = `${await getApiUrl()}/api/user/acknowledgements`;
+    const res = await axios.get(url, Config.authOpts());
+    return res.data;
+  },
+  acceptAcknowledgments: async (...keys) => {
+    if (keys.length === 0) {
+      return {};
+    }
+
+    const url = `${await getApiUrl()}/api/user/acknowledgements`;
+    const res = await fetchOk(url, fp.mergeAll([Config.authOpts(), Config.jsonBody(keys), { method: 'POST' }]));
```

Comment:
I'm not sure if we've documented this anywhere, but we did start moving toward using `axios` over `fetch`. Try `axios.post` instead:

```suggestion
    const res = await axios.post(url, Config.jsonBody(keys), Config.authOpts());
```